client/src/features/modal/EventModal.jsx

```diff
@@ -9,6 +9,36 @@ import dataService from "../../services/dataService";
 import { useAuthContext } from "../../contexts/AuthContext";
 import { useEventsContext } from "../../contexts/EventsContext";
 
+const URLRegex = /(https?:\/\/[^\s]+|www\.[^\s]+)/g;
+
+const LinkText = ({ children }) => {
+  if (typeof children !== "string")
+    throw new Error("LinkText can only accept a string as children");
+
+  const separated = children.split(URLRegex);
+
+  return separated.map((res, index) => {
+    if (URLRegex.test(res)) {
+      // Prepend http:// if it starts with www. to make a valid href
+      const href = res.startsWith("www.")
+        ? `http://${res}`
```

Comment:
```suggestion
        ? `https://${res}`
```
i'm thinking we would want to add `https` instead of just `http`?

Reply:
Good catch, I'll try to have this done tomorrow.